lldb/source/Target/Target.cpp

```diff
@@ -1476,8 +1476,26 @@ static void LoadTypeSummariesForModule(ModuleSP module_sp) {
   auto section_size = summaries_sp->GetSectionData(extractor);
   lldb::offset_t offset = 0;
   while (offset < section_size) {
+    // Skip null bytes. Can happen with alignment padding.
+    while (true) {
+      auto next_offset = offset;
+      if (extractor.GetU8(&next_offset) != 0) {
```

Comment:
Why is next_offset needed? Does GetU8 modify the parameter in the failure case?

Reply:
yes, the passed in offset is an in-out parameter. As far as `GetU8` is concerned, reading a zero byte is not a failure case. If there was a `PeekU8` method that would be better suited.

Reply:
open to suggestions